tests/regression/56-floats/06-library_functions.c

```diff
@@ -0,0 +1,40 @@
+// PARAM: --enable ana.int.interval --enable ana.int.def_exc --enable ana.float.interval
+#include <assert.h>
+#include <math.h>
+
+int main() {
+    double dbl_min = 2.2250738585072014e-308;

```

Comment:
include `float.h` and use `DBL_MIN` instead of this magic constant

Reply:
No, because then the regression test does not work anymore

Reply:
for some reason, if I do that, goblint thinks that DBL_MIN = 0 (that was also the reason for the debug prints in FloatDomain -> signbit)

Reply:
Why? `DBL_MIN` should give you exact the smallest normal number. That's what you are looking for, isn't it?

Reply:
This does not seem right...
When i try, i get this warning `treating long double constant 2.2250738585072014e-308L as double constant` but can't find where it is actually coming from. However, this could explain the behaviour. But we should either fix this or get a deeper understanding of what this is doing, before we continue...

Reply:
This [commit](https://github.com/Dudeldu/analyzer/pull/10/commits/715d6ae2e10d3e21fb93fb53d8a40ce7762c2244) should provide a solution, but is not really satisfying. But i see no other option

Reply:
With #10 merged, we should now be able to use DBL_MIN